src/components/ImageComponent/ImageComponent.js

```diff
@@ -1,11 +1,14 @@
 import { observerListener } from '../../shared/observer';
 
 const template = (imageUrl, className, id) => {
-  return `
-    <img src="${imageUrl}" class="${className}" id="${id}"/>
-  `
+  let src = true;
```

Comment:
1. If(src)? 
You just setted up it to true, this condition would always be true
2. You forget to wrap up the statement in return with ``
3. You can go just with in src in img

imgUrl || 'link to placeholder'

Reply:
changed it